src/settings.rs

```diff
@@ -6,7 +6,11 @@ use crate::{
 };
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
```

Comment:
remove this

Reply:
> remove this

Sorry, I did not get this. What needs to be removed?

Reply:
My bad, it thought it was not used anymore! You can leave it :)